### DISCUSSION TITLE
Why create project with comand npx create-pixi3d-app@latst my-pixi3d-app, after start notwork?

### DISCUSSION THREAD

**Discussion by gilzero04** [jnsmalm/pixi3d#189] [General]
Why create project with comand npx create-pixi3d-app@latst my-pixi3d-app, after start notwork?

![image](https://github.com/jnsmalm/pixi3d/assets/50231337/a11a7f77-ca40-47e5-829d-fe91c0503229)

How to solve this problem?

Thank you very much.

**Comment by jnsmalm** [comment]
Thanks for reporting!

I think I have fixed this now, try running the command `npx create-pixi3d-app@latest my-pixi3d-app` again.

Problem is that something has changed with the types after PixiJS v7.3. I need to look into this later, but for now the above command will use PixiJS v7.2.4 instead of latest.

**Reply by gilzero04** [reply]
Thank you very much.
